cmd/clusterctl/cmd/root.go

```diff
@@ -67,6 +67,11 @@ var RootCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
+		disable, err := configClient.Variables().Get("DISABLE_VERSIONCHECK")
```

Comment:
```suggestion
		disable, err := configClient.Variables().Get("CLUSTERCTL_DISABLE_VERSIONCHECK")
```
WDYT about prefixing this with CLUSTERCTL_ (like with the log level)